contrib/run_function.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+export KUBECONFIG="$(kind get kubeconfig-path)"
+
+kubectl rollout status deploy/gateway -n openfaas --timeout=0s
```

Comment:
This is not what I asked for. I assume timeout=0s means infinite? We definitely need at timeout, set it to something sensible and we can tune it over time.

Reply:
Added "1m" as timout.